requirements.txt

```diff
@@ -1,4 +1,5 @@
 boto3
+kombu==4.6.3
```

Comment:
This fixes the issue found in https://issues.apache.org/jira/browse/AIRFLOW-5240

Reply:
@emtwo already filed https://github.com/mozilla/telemetry-airflow/pull/598 separately for this. I'm merging that one.

Reply:
Merged. You'll need to rebase.